src/providers/Asset.tsx

```diff
@@ -14,14 +14,13 @@ import axios, { CancelToken } from 'axios'
 import { retrieveDDO } from '../utils/aquarius'
 import { getPrice } from '../utils/subgraph'
 import { MetadataMarket } from '../@types/MetaData'
-import { DDO_TEMPORARY } from './Ocean'
 import { useWeb3 } from './Web3'
 import { useSiteMetadata } from '../hooks/useSiteMetadata'
 
 interface AssetProviderValue {
   isInPurgatory: boolean
   purgatoryData: PurgatoryData
-  ddo: DDO
+  ddo: any
```

Comment:
same here we can add back the DDO type, i think you can create a new PR with putting back the DDO this changes and merge it when we have a new version of oceanjs released